pageserver/src/page_service.rs

```diff
@@ -2505,6 +2505,52 @@ impl PageServiceCmd {
     }
 }
 
+fn parse_options(options: &str) -> (Vec<(String, String)>, bool) {
+    let mut parsing_config = false;
+    let mut has_error = false;
+    let mut config = Vec::new();
```

Comment:
Did you choose vec because you don't expect many options?

Reply:
Yep I want the parse function just append everything it parsed and let the caller decide how to deal with duplicates.